kratos/geometries/line_3d_2.h

```diff
@@ -403,6 +403,26 @@ class Line3D2 : public Geometry<TPointType>
     }
 
 
+    ///@}
+    ///@name Intersections
+    ///@{
+
+    /**
+     * @brief Test if this geometry intersects with other geometry
+     * @param  ThisGeometry Geometry to intersect with
+     * @return              True if the geometries intersect, False in any other case.
+     */
+    bool HasIntersection(const BaseType& rThisGeometry) const override {
```

Comment:
```suggestion
    bool HasIntersection(const BaseType& rThisGeometry) const override
    {
```
(minor style guide suggestion)